recipes/libgpuarray/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: libgpuarray
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/Theano/libgpuarray.git
+  git_tag: v0.6.0
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - git
+    - cmake
+    - python
+    - cython >=0.25
+    - numpy
```

Comment:
Does this link to NumPy? If so, we need to add `x.x` after it. Also that will be needed in `run` too.